TheFourFifths/consus-client
# Feature/tff 56 check out/ui

### Summary

Working on UI for check out flow

### Checklist


Finally, is it ready to be reviewed and merged: 📗  

### How to Review

1. Run the current server on the dev branch of consus
2. Build and run the electron client
3. Type the student id (123456) into the omni-search bar
4. **Don't** hit the search button. It literally does nothing for this task
6. Verify that the Student view pops up displaying a student, their id, and a message saying they have no items checked out.  A cart view should also appear with a search bar.
5. Enter the item id (iGwEZUvfA).  It will be added to the cart.
7. Add another item id (iGwEZVHHE). It will also be added.
8. Click the checkout button.  The cart will empty and the items are added to the student's checked-out items.

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-56](https://msoese.atlassian.net/browse/TFF-56)
